src/lib/actions/watchAnnouncementsForUser/watchAnnouncementsForUser.test.ts

```diff
@@ -1,14 +1,19 @@
 import { describe, test, expect, afterAll, beforeAll } from 'bun:test';
 import setupTestEnv from '../../helpers/test/setupTestEnv';
 import {
-  type AnnouncementArgs,
   type AnnouncementLog,
   ERC5564AnnouncerAbi,
   VALID_SCHEME_ID,
   generateStealthAddress,
 } from '../../..';
 import setupTestWallet from '../../helpers/test/setupTestWallet';
 import setupTestStealthKeys from '../../helpers/test/setupTestStealthKeys';
+import type { StealthActions } from '../../stealthClient/types';
+import type { Address } from 'viem';
+import { type SuperWalletClient } from '../../helpers/types';
+
+const NUM_ACCOUNCEMENTS = 3;
```

Comment:
Here and other instances/mention of this var.
```suggestion
const NUM_ANNOUNCEMENTS = 3;
```

Reply:
https://github.com/ScopeLift/stealth-address-sdk/pull/54/commits/4ccac661018f287d2928dab2175955a6ac802eea